### DISCUSSION TITLE
Error when I make a second move at main

### DISCUSSION THREAD

**Discussion by DelusionArun** [CastleLab/COMP3021-2021Fall-PA1-Student-Version#16] [Q&A] [answered]
I wanted to try debugging at Main and while the first move works fine, when I make a second move (no mater valid, invalid, return etc.), I get -
Execution failed for task ':app:Main.main()'.
> Process 'command 'C:/Program Files/Java/jdk-16.0.2/bin/java.exe'' finished with non-zero exit value 1

According to stackoverflow, this usually occurs when main cannot locate file because of gradle/plugin incompatibility. Is it the likely case here, or can this error occur because of my wrong code as well?

**Comment by Derppening** [comment, marked as answer]
It can be because of your wrong code.

To see the full error message, click on where it says `:app:Main.main()`, for example:

![thumbnail_image](https://user-images.githubusercontent.com/4293438/134804060-69ff891d-2b28-46ab-bef9-cf3e2f02b74f.png)

In this case, that is because I forgot to pass the map file. Your error message might be different, and perhaps you can start from there.

**Reply by DelusionArun** [reply]
Yes, that is the exact error I have as well.  
Is passing the map file relevant to a particular java file (i.e, something we need to implement)? or do I just need to update some skeleton code? 
Thank you for your help.

**Reply by Derppening** [reply]
That part has already been implemented for you.

Perhaps there is [a section in the README](https://github.com/CastleLab/COMP3021-2021Fall-PA1-Student-Version#intellij-idea) that could help with that?

![image](https://user-images.githubusercontent.com/4293438/134804415-880c7244-c19c-482c-8bdd-f4b75cfe42f5.png)
